com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/SessionController.java

```diff
@@ -90,15 +90,15 @@ public void modelChanged(String modeluri, CCommand command) {
 		// TODO: Distinguish from wholesale update?
```

Comment:
Can we remove this TODO now?

Reply:
Yes indeed! Will do with https://github.com/eclipsesource/modelserver/issues/87